darts/models/forecasting/croston.py

```diff
@@ -0,0 +1,61 @@
+"""
+Croston method
+--------------
+"""
+
+import numpy as np
+from statsforecast.models import croston_classic, croston_optimized
+
+from darts.models.forecasting.forecasting_model import ForecastingModel
+from darts.timeseries import TimeSeries
+
+
+class Croston(ForecastingModel):
```

Comment:
Apparently TSB and croston_sba are improved version of Croston (https://www.lancaster.ac.uk/pg/waller/pdfs/Intermittent_Demand_Forecasting.pdf), might be worth proposing each of the 3 options in the same function as they already exist in statsforecast?

Reply:
Good idea, done.